### ISSUE TITLE
The reproduced experiments of S3DIS Area5 segmentation can't reach 71.4 mIoU

### ISSUE THREAD

**Issue by linhaojia13** [LifeBeyondExpectations/ECCV22-PointMixer#11]
Hi, dear authors.
According to the script `run_s3dis_PointMixer.sh`, I train the PointMixer on the S3DIS and evaluate it in the setting of `npts=800k`. The mIoU of the evaluation only reach 69.11, which has a dominant gap with the reported results in the paper. Did I miss something and bring about this gap?

```
Val result: mIoU/mAcc/allAcc 0.6911/0.7513/0.8989.
Val1 result: mIoU/mAcc/allAcc 0.6911/0.7513/0.8989.
Class_0 Result: iou/accuracy 0.9263/0.9576, name: ceiling.
Class_1 Result: iou/accuracy 0.9798/0.9844, name: floor.
Class_2 Result: iou/accuracy 0.8433/0.9619, name: wall.
Class_3 Result: iou/accuracy 0.0000/0.0000, name: beam.
Class_4 Result: iou/accuracy 0.3351/0.3756, name: column.
Class_5 Result: iou/accuracy 0.6059/0.6212, name: window.
Class_6 Result: iou/accuracy 0.7650/0.9073, name: door.
Class_7 Result: iou/accuracy 0.8343/0.8907, name: chair.
Class_8 Result: iou/accuracy 0.8986/0.9528, name: table.
Class_9 Result: iou/accuracy 0.7628/0.7992, name: bookcase.
Class_10 Result: iou/accuracy 0.7230/0.7979, name: sofa.
Class_11 Result: iou/accuracy 0.7349/0.7707, name: board.
Class_12 Result: iou/accuracy 0.5753/0.7477, name: clutter.
<<<<<<<<<<<<<<<<< End Evaluation <<<<<<<<<<<<<<<<<
Shutting down background jobs, please wait a moment...
Done!
Waiting for the remaining 82 operations to synchronize with Neptune. Do not kill this process.
All 82 operations synced, thanks for waiting!
Explore the metadata in the Neptune app:
https://app.neptune.ai/jaesung.choe/ECCV22-PointMixer-SemSeg/e/EC-182
```

**Comment by LifeBeyondExpectations** [comment]
Regarding the reproducibility, this problem is related to the scatter_softmax function.

Simply, when you train [Point Transformer based on our implementation](https://github.com/LifeBeyondExpectations/ECCV22-PointMixer/blob/bf1c3fb7187d737d68901e7165e78d64ab3abe4d/sem_seg/script/run_s3dis_PointTransformer.sh), 
you can get the exact same performance for every training phase.
However, this is not the same for PointMixer due to the un-determinstic gradient computation in the [scatter_softmax(...)](https://github.com/LifeBeyondExpectations/ECCV22-PointMixer/blob/bf1c3fb7187d737d68901e7165e78d64ab3abe4d/sem_seg/model/network/inter.py#L83).

Thus, the same hyper-parameter setup can result in various performance.

We did not yet upload the pretrained weights, 
but sometimes, we achieve even higher performance under the same hyper-parameters.

**Comment by linhaojia13** [comment]
Thank you for your detailed reply. I am looking forward to the pretrained weights!
